library/spdm_responder_conformance_test_lib/spdm_responder_test_12_heartbeat_ack.c

```diff
@@ -409,15 +409,15 @@ void spdm_test_case_heartbeat_ack_unexpected_request (void *test_context)
         common_test_record_test_assertion (
             SPDM_RESPONDER_TEST_GROUP_MEASUREMENTS,
```

Comment:
Is `SPDM_RESPONDER_TEST_GROUP_MEASUREMENTS` correct?
I think `SPDM_RESPONDER_TEST_GROUP_HEARTBEAT_ACK` is correct.

Reply:
You are right. This is a bug.